src/components/SampleApp/index.tsx

```diff
@@ -33,22 +32,20 @@ import { compressToBase64, decompressFromBase64 } from 'lz-string';
  *
  */
 import { routes } from '@wormhole-foundation/sdk';
-import {
-  MayanRoute,
-  MayanRouteWH,
-  MayanRouteMCTP,
-  MayanRouteSWIFT,
-  MayanRouteMONOCHAIN,
-} from '@mayanfinance/wormhole-sdk-route';
-import { NTT_TEST_CONFIG_TESTNET, NTT_TEST_CONFIG_MAINNET } from './consts';
-import { DEFAULT_ROUTES } from 'routes/operator';
-import { nttRoutes } from 'exports/ntt';
 import {
   cctpExecutorRoute,
-  cctpV2StandardExecutorRoute,
   cctpV2FastExecutorRoute,
+  cctpV2StandardExecutorRoute,
 } from 'exports/executor';
+import { nttRoutes } from 'exports/ntt';
+import { MayanRoute } from 'routes/mayan/MayanRoute';
```

Comment:
Maybe worth having a `routes/mayan/index.ts` file, so that we just `import { ... } from "routes/mayan"`